tests/v2v_test.go

```diff
@@ -0,0 +1,71 @@
+package tests_test
+
+import (
+	"log"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	"kubevirt.io/kubevirt-ansible/tests"
+)
+
+// template parameters
+const (
+	v2vServiceInstance    = "tests/manifests/import-vm-v2v-apb.yml"
+	dstv2vServiceInstance = "/tmp/test-v2v.json"
+)
+
+// Array of enviroment variables to get from testing env
+var listofEnvVars = []string{"ocpID", "ocpPass", "VMWareURL", "VMName", "VMWareID", "VMWarePass"}
+
+// Map to store Enviroment variables
+var v2vEnvVars map[string]string
+
+// Get all the Env variable needed for tests
+func getV2vEnv() {
+	v2vEnvVars = map[string]string{}
+	for _, vars := range listofEnvVars {
+		val, ok := os.LookupEnv(vars)
+		if !ok {
+			//raise exception
+			log.Fatal("Expected env var not fond: " + vars)
+		}
+		v2vEnvVars[vars] = val
+	}
+}
+
+var _ = Describe("Importing VMs using V2V", func() {
+
+	BeforeEach(func() {
+		getV2vEnv()
+	})
+
+	Context("Create ServiceInstance of import-vm-apb with vmware plan", func() {
+		It("should import vm suceessfully", func() {
+			tests.ProcessTemplateWithParameters(v2vServiceInstance, dstv2vServiceInstance,
```

Comment:
44-52 and 59-67 are exact same which can be reduced by either puting those code into a `BeforeEach` or `JustBeforeEach` block or using `table.Entry`

Reply:
https://github.com/kubevirt/kubevirt-ansible/blob/b5e629df47c2d733f54c426052f072eab07f4c4e/tests/v2v_test.go#L45
https://github.com/kubevirt/kubevirt-ansible/blob/b5e629df47c2d733f54c426052f072eab07f4c4e/tests/v2v_test.go#L60
@shiywang the param `APB_PLAN_NAME` is different in both, so it is not repeating same thing
Can you suggest a way to parametrize it?

Reply:
can't you put those two into `v2vEnvVars`? if not another way is define a local var like `apbPlanName`, then both will be  `"APB_PLAN_NAME="+apbPlanName,`

Reply:
@shiywang @nellyc added use of Ginkgo table to parametrize tests from single Describe